src/headless-cms/components/cmsSidebarContent/CmsSidebarContentLayoutLinkList.tsx

```diff
@@ -0,0 +1,67 @@
+import { IconArrowRight } from 'hds-react';
+import React from 'react';
+
+import ExternalLink from '../../../common/components/externalLink/ExternalLink';
+import HtmlToReact from '../../../common/components/htmlToReact/HtmlToReact';
+import styles from './cmsSidebarContentLayoutLinkList.module.scss';
+
+type Link = {
+  target?: string | null;
+  title?: string | null;
+  url?: string | null;
+};
+
+type Props = {
+  anchor?: string | null;
+  title?: string | null;
+  description?: string | null;
+  links?: Array<{
+    target?: string | null;
+    title?: string | null;
+    url?: string | null;
+  } | null> | null;
+};
+
+const CmsSidebarContentLayoutLinkList: React.FC<Props> = ({
+  anchor,
+  title,
+  description,
+  links,
+}) => {
+  return (
+    // I assumed that the anchor references the ID that an anchor element may
+    // target with href="#anchor"
+    <div id={anchor || undefined}>
```

Comment:
What is this used for?

Reply:
Honestly, I'm not sure. My assumption is, because of the data model, that it's an ID that can be used create anchor links.

The data model seems to be more suited for creating sections. In which case linking to them makes more sense. E.g. you could have a link within content that links to a section later on ("Check out this content").

Here it doesn't make as much sense, but I added it to be complete in case it's not in this data model by coincidence, but because there's actual some requirement it checks.